projects/metis/src/app/dataset/executions-data-grid/executions-data-grid.component.ts

```diff
@@ -19,7 +19,8 @@ import {
 })
 export class ExecutionsDataGridComponent {
   @Input() hasMore?: boolean;
-
+  @Input() applyHighlight?: boolean;
+  @Input() applyStripe?: boolean;
```

Comment:
I agree on `applyHighlight` - it could be contained within the component and doesn't need the `Input` decorator, but I do think `applyStripe` is a more descriptive name than `indexInWorkflow`.